src/frontend/planner_test/tests/testdata/agg.yaml

```diff
@@ -190,39 +190,39 @@
   batch_plan: |
     BatchExchange { order: [], dist: Single }
     └─BatchSortAgg { group_key: [mv.v1], aggs: [max(mv.v2)] }
-      └─BatchExchange { order: [mv.v1 DESC], dist: HashShard(mv.v1) }
+      └─BatchExchange { order: [mv.v1 DESC NULLS FIRST], dist: HashShard(mv.v1) }
```

Comment:
Just an idea: maybe we can omit `NULLS FIRST` (default value) to make the explain more compact (and avoid merge conflict..). But being explict can also be good, so I don't have strong opinion

Reply:
FWIW, PG's:
```
xxchan@/tmp:postgres> explain select * from t order by x desc nulls last;
+-----------------------------------------------------------+
| QUERY PLAN                                                |
|-----------------------------------------------------------|
| Sort  (cost=179.78..186.16 rows=2550 width=4)             |
|   Sort Key: x DESC NULLS LAST                             |
|   ->  Seq Scan on t  (cost=0.00..35.50 rows=2550 width=4) |
+-----------------------------------------------------------+
EXPLAIN 3
Time: 0.012s
xxchan@/tmp:postgres> explain select * from t order by x desc nulls first;
+-----------------------------------------------------------+
| QUERY PLAN                                                |
|-----------------------------------------------------------|
| Sort  (cost=179.78..186.16 rows=2550 width=4)             |
|   Sort Key: x DESC                                        |
|   ->  Seq Scan on t  (cost=0.00..35.50 rows=2550 width=4) |
+-----------------------------------------------------------+
EXPLAIN 3
Time: 0.007s
xxchan@/tmp:postgres> explain select * from t order by x asc nulls last;
+-----------------------------------------------------------+
| QUERY PLAN                                                |
|-----------------------------------------------------------|
| Sort  (cost=179.78..186.16 rows=2550 width=4)             |
|   Sort Key: x                                             |
|   ->  Seq Scan on t  (cost=0.00..35.50 rows=2550 width=4) |
+-----------------------------------------------------------+
EXPLAIN 3
Time: 0.002s
xxchan@/tmp:postgres> explain select * from t order by x asc nulls first;
+-----------------------------------------------------------+
| QUERY PLAN                                                |
|-----------------------------------------------------------|
| Sort  (cost=179.78..186.16 rows=2550 width=4)             |
|   Sort Key: x NULLS FIRST                                 |
|   ->  Seq Scan on t  (cost=0.00..35.50 rows=2550 width=4) |
+-----------------------------------------------------------+
EXPLAIN 3
```

Reply:
Actually I tried to hide it if it's default value, but then I found that if it's set by user, not printing it in explain string may make user confused, so I decided to keep it explicit.

Reply:
Anyway, both LGTM. Since PG hides default value, we can do it as well then😄